blog/2022-01-bm4-update/README.md

```diff
@@ -0,0 +1,482 @@
+---
+title: 'Den Book-Monkey v4 updaten (3. Ausgabe)'
+author: Angular Buch Team
+mail: team@angular-buch.com
+published: 2022-01-01
+keywords:
+  - Angular
+  - Angular 12
+  - Angular 13
+  - Strict Mode
+language: de
+thumbnail: ./angular12.jpg
+---
+
+Das Angular-Ökosystem wird kontinuierlich verbessert.
+Das Release einer neuen Major-Version von Angular bedeutet keineswegs, dass alle Ideen verworfen werden und Ihre Software nach einem Update nicht mehr funktioniert.
+Die Grundideen von Angular sind seit Version 2 konsistent und auf Beständigkeit über einen langen Zeitraum ausgelegt.
+Die in unserem Buch beschriebenen Konzepte behalten ihre Gültigkeit.
+
+Ein paar kleine Änderungen haben sich jedoch seit der Veröffentlichung der 3. Ausgabe unseres Buchs ergeben.
+Diese wollen wir hier detailiert besprechen.
+Es geht vor allem daraum, dass seit **Angular 12** diverse strikte Einstellung für neue Projekte standardmäßig aktiviert sind.
+Als wir das Buch im Oktober 2020 veröffentlicht haben, war das noch nicht so.
+Sind die strikten Einstellungen aktiv, brechen nun leider einige gedruckte Beispiele, die sich aber mit moderatem Aufwand beheben lassen.
+
+## Der BookMonkey
+
+Der "BookMonkey" ist das Demo-Projekt zum Buch.
+Anhand des Beispielprojekts führen wir Sie schrittweise an die Entwicklung mit Angular heran.
+
+Alle Entwicklungsschritte im Buch stellen wir in separaten Repositorys zur Verfügung.
+Wenn man den Anleitungen im Buch folgt, sieht die eigene Codebasis im Idealfall genauso aus, wie unser Stand auf GitHub.
+
+
+## Einen bestehenden BookMonkey updaten
+
+Wenn Sie unser Buch gleich nach der Veröffentlichung gekauft haben und alle Beispiele daraufhin nach Anleitung umgesetzt haben, dann haben Sie keinen großen Aufwand.
+Zum Zeitpunkt der Veröffentlichung war Angular 10 die neueste Version, kurz danach folgte Angular 11.
+Wurde Ihr BookMonkey in dieser Zeit erstellt, dann sind in ihrem Projekt noch keinen strikten Einstellungen aktiv.
+
+Sie können die verwendete Angular-Version in der Datei `package.json` ablesen.
+Der Befehl `ng version` liefert ebenfalls ausführliche Infos zur Angular-Version im jeweiligen Projekt.
+
+Um auf den neuesten Stand von Angular zu gelangen, benutzen Sie bitte den Befehl `ng update` in der Kommandozeile und folgen Sie den Anweisungen auf dem Bildschirm.
+
+Lesen Sie dazu auch gerne unsere Blogposts mit den neuesten Änderungen zu Angular:
+
+* [Angular 11 ist da!](/blog/2020-11-angular11)
+* [Angular 12 ist da!](/blog/2021-05-angular12)
+* [Angular 13 ist da!](/blog/2021-11-angular13)
+
+
+## Einen neuen BookMonkey erstellen
+
+Wenn Sie heute wie im Buch beschrieben den BookMonkey mit `ng new` erzeugen, so wird das Projekt standardmäßig mit strikten Einstellungen erstellt.
```

Comment:
Heute sollte raus. Wenn das jemand in 5 Jahren liest passt das nicht mehr

Reply:
Darüber könnte man diskutieren. Die strikten Einstellungen gehen bestimmt nicht weg...

Reply:
Dann lasst uns doch den relativen zeitlichen Bezug durch eine absolute Angabe ersetzen:
> Wenn Sie nach Juni 2021 wie im Buch beschrieben den BookMonkey mit `ng new` erzeugen,

Reply:
ohne "heute"

Reply:
... oder so!